src/paperless/utils.py

```diff
@@ -0,0 +1,9 @@
+import unicodedata
+
+
+def slugify(content):
```

Comment:
Can we come up with a more appropriate name for this than `slugify()`?  Something like `make_searchable()` would be more apt as this doesn't turn the content into a slug at all.